src/backend/marsha/e2e/test_lti.py

```diff
@@ -331,6 +336,156 @@ def test_lti_select(page: Page, live_server: LiveServer):
+                        "url": f"{live_server.url}/lti/documents/{document.id}",
+                        "frame": [],
+                        "title": f"{lti_consumer_parameters.get('title')}",
+                        "text": f"{document.description}",
+                    }
+                ],
+            }
+        )
+        .replace(", ", ",")
+        .replace(": ", ":")
+    )
+    assert document_content_items not in lti_select_iframe.content()
+    with page.expect_request("**/lti/respond/"):
+        lti_select_iframe.click(f'[title="Select {document.title}"]')
+    lti_select_iframe.wait_for_selector("dd")
+    assert document_content_items in lti_select_iframe.content()
+
+    # Select a video
+    page.click('#lti_select input[type="submit"]')
+    lti_select_iframe.click('button[role="tab"]:has-text("Videos")')
+    video_content_items = (
+        json.dumps(
+            {
+                "@context": "http://purl.imsglobal.org/ctx/lti/v1/ContentItem",
+                "@graph": [
+                    {
+                        "@type": "ContentItem",
+                        "url": f"{live_server}/lti/videos/{video.id}",
+                        "frame": [],
+                        "title": f"{lti_consumer_parameters.get('title')}",
+                        "text": f"{video.description}",
+                    }
+                ],
+            }
+        )
+        .replace(", ", ",")
+        .replace(": ", ":")
+    )
+    assert video_content_items not in lti_select_iframe.content()
+    with page.expect_request("**/lti/respond/"):
+        lti_select_iframe.click(f'[title="Select {video.title}"]')
+    lti_select_iframe.wait_for_selector("dd")
+    assert video_content_items in lti_select_iframe.content()
+    assert Video.objects.count() == 1
+
+    # Select a new video
+    page.click('#lti_select input[type="submit"]')
+    lti_select_iframe.click('button[role="tab"]:has-text("Videos")')
+    sent_title = f'"title":"{lti_consumer_parameters.get("title")}"'
+    assert sent_title not in lti_select_iframe.content()
+    with page.expect_request("**/lti/respond/"):
+        lti_select_iframe.click("text=Add a video")
+    lti_select_iframe.wait_for_selector("dd")
+
+    # added video is created
+    assert Video.objects.count() == 2
+    added_video = Video.objects.exclude(id=video.id).first()
+    assert added_video.title == lti_consumer_parameters.get("title")
+    assert added_video.description == lti_consumer_parameters.get("text")
+    video_content_items = (
+        json.dumps(
+            {
+                "@context": "http://purl.imsglobal.org/ctx/lti/v1/ContentItem",
+                "@graph": [
+                    {
+                        "@type": "ContentItem",
+                        "url": f"{live_server}/lti/videos/{added_video.id}",
+                        "frame": [],
+                        "title": f"{lti_consumer_parameters.get('title')}",
+                    }
```

Comment:
a small comment to ease the reading of the test
the video is created and there is no description...